src/intentManager.js

```diff
@@ -219,70 +154,38 @@ define([
         icon: 'icon-vellum-android-intent',
         isODKOnly: true,
         isTypeChangeable: false,
-        intentTag: null,
         init: function (mug, form) {
+            mug.p.intentXmlns = mug.p.intentXmlns || DEFAULT_XMLNS;
         },
         spec: {
             androidIntentAppId: {
                 lstring: 'Intent ID',
                 visibility: 'visible',
                 widget: androidIntentAppId,
-                serialize: function (value, key, mug, data) {
```

Comment:
Yes, it's officially unsupported, but that doesn't mean people will not report bugs if things don't work the way they expect.

I made a quick attempt at preserving the old serialization format: fa41fa2923da949ab9f8ee059d34804870a3c7fe. You can cherry-pick it if you like it. I won't be offended if you think preserving backward compatibility is not worth the added serialization complexity.